mintlify/pay-to-bank/developer-guides/platform-configuration.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: "Platform Configuration"
+description: "Configuring Pay to bank settings"
```

Comment:
Let's hold off on the pay to bank stuff in this PR. We can get to that as @exs6350 sorts out the API schema